js/meetup/App.js

```diff
@@ -46,7 +48,7 @@ function SlideAndVideo(props) {
   if (props.session && (slides_link || video_link || demo_link)) {
     return (<div className="slide-and-video">
       {slides_link && <a href={slides_link} target="_blank" rel='noopener noreferrer'><i className="icon-file-powerpoint-o"></i>  Slide</a>} &nbsp;&nbsp;
-      {video_link && <a href={video_link} target="_blank" rel='noopener noreferrer'><i className="icon-youtube-play"></i>  Video</a>} &nbsp;&nbsp;
+      {video_link && <Player session={props.session} width="76em" height="42em"/>} &nbsp;&nbsp;
```

Comment:
Since this has fixed value, it won't be responsive?

Reply:
Responsiveness added for player. Addressed.